packages/react-instantsearch/src/core/findResultsState.js

```diff
@@ -0,0 +1,16 @@
+/** 
```

Comment:
Legacy js doc module issues, we will have to replace it as soon as it will get too much in our way (its parser is not really comfortable with recent ways of coding).